static/app/components/projectPageFilter.tsx

```diff
@@ -80,7 +80,7 @@ type Props = {
 
 function ProjectPageFilter({
   specificProjectSlugs,
-  maxTitleLength = 30,
+  maxTitleLength = 25,
```

Comment:
Decreased this from 30 to 25 because it was taking up too much of the maximum `30rem` width allowed by the page filter bar.